src/components/TodoField/TodoField.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useContext, useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoContext } from '../../contexts/TodoContext';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoField: React.FC<Props> = ({ todo }) => {
+  // console.log('render Todo ' + todo.id);
```

Comment:
GH thats the place where your code that's ready for review should be perfectly clear ;)